docs/source/quick_start/troubleshooting.rst

```diff
@@ -0,0 +1,130 @@
+
+Troubleshooting
+===============
+
+Have problems making things work? Here are some common issues and their solutions
+
+MCDR won't start
+----------------
+
+* **mcdreforged: command not found**
+    * Try ``python -m mcdreforged``
```

Comment:
```suggestion
    * Try ``python -m mcdreforged``, which explicitly specifies the Python interpreter to start MCDR
```

Reply:
Another question, if MCDR is installed via pipx, does this solution still work?

Reply:
idk, and Ii didn't find anything about that in the pipx doc either

Reply:
I think it does not work, cuz the `mcdreforged` package wont be inside any site-packages that the python interpreter started from the `python` command can access

Quick test from currently latest ubuntu docker image, with the following commands

```bash
apt update && apt install pipx
pipx ensurepath
bash -l
pipx install mcdreforged
python3 -m mcdreforged
```

Reply:
Maybe you can add some statement on `pipx ensurepath` abd `bash -l` (to ensure the PATH is correctly configured from `.bashrc` or whatever file), instead of `python -m`, for pipx

Reply:
I think the current statement is enough. pipx prints a warning when there ensurepath not executed, and users should refer to the pipx documentation when encountering problems